lighthouse-core/audits/performance-budget.js

```diff
@@ -0,0 +1,157 @@
+/**
+ * @license Copyright 2019 Google Inc. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+'use strict';
+
+const Audit = require('./audit.js');
+const ResourceSummary = require('../computed/resource-summary.js');
+const i18n = require('../lib/i18n/i18n.js');
+
+const UIStrings = {
+  /** Title of a Lighthouse audit that compares the size and quantity of page resources against targets set by the user. These targets are thought of as "performance budgets" because these metrics impact page performance (i.e. how quickly a page loads). */
+  title: 'Performance budget',
+  /** Description of a Lighthouse audit where a user sets budgets for the quantity and size of page resources. No character length limits. */
+  description: 'Keep the quantity and size of network requests under the targets ' +
+    'set by the provided performance budget.',
+  /** [ICU Syntax] Label identifying the number of requests over the budget for requests*/
+  requestCount: `{count, plural,
```

Comment:
```suggestion
  countOverBudgetEntry: `{count, plural,
```
maybe? Kind of confusing with the other use of `requestCount` in this file